otherlibs/stdlib_alpha/capsule.ml

```diff
@@ -302,10 +302,12 @@ module Mutex = struct
     ; mutex : M.t
     ; mutable poisoned : bool
     }
+  [@@unsafe_allow_any_mode_crossing]
 
-  (* CR: illegal mode crossing on the current version of the compiler,
-     but should be legal. *)
   type packed : value mod portable uncontended = P : 'k t -> packed
+  [@@unsafe_allow_any_mode_crossing
+    "CR: illegal mode crossing on the current version of the compiler, but \
```

Comment:
```suggestion
    "CR layouts v2.8: illegal mode crossing on the current version of the compiler, but \
```
This will make it easier to find later.